src/main/resources/templates/DownloadResource/download.html

```diff
@@ -0,0 +1,154 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <!--<meta http-equiv="refresh" content="0; url={htmlData.getDownloadLink()}">-->
+    <title>Download</title>
+</head>
+<body>
+<div class="banner lead">
+    Adoptium
+</div>
+
+<div class="container">
+    <div class="left-column">
+        <h2>{msg:download_starting}</h2>
+        <p>Thank you for downloading</p>
+        <a href="{htmlData.getDownloadLink()}">
+            {htmlData.getVendor()} {htmlData.getImageType()} {htmlData.getVersion()}<br>
+            for {htmlData.getOs()} {htmlData.getArch()}
+        </a>
+        <button id="btn-copy-dll">Copy</button>
+        <p>
+            If the downlaod does not start within 10 seconds,<br>
+            <a href="{htmlData.getDownloadLink()}">try again to download manually</a>
+        </p>
+        <p>checksum (sha256): {htmlData.getChecksum()}</p>
+        <button id="btn-copy-chksm">Copy</button>
+    </div>
+</div>
+<style type="text/css" scoped>
+    h1, h2, h3, h4, h5, h6 {
```

Comment:
Can we externalise CSS in a separate file?

Reply:
CSS externalized in own file index.css